static/js/actions/ui.js

```diff
@@ -65,3 +65,18 @@ export const SET_USER_PAGE_DIALOG_VISIBILITY = 'SET_USER_PAGE_DIALOG_VISIBILITY'
 export const setUserPageDialogVisibility = bool => (
   { type: SET_USER_PAGE_DIALOG_VISIBILITY, payload: bool }
 );
+
+export const SET_SHOW_EDUCATION_DELETE_DIALOG = 'SET_SHOW_EDUCATION_DELETE_DIALOG';
```

Comment:
This is because the sequence looked something like this:

- user clicks 'delete' icon on a work history entry
- dispatch `SET_DELETION_INDEX`
- dispatch `SET_SHOW_CONFIRM_DELETE_DIALOG`
- education and work history popups both display (they both have the `open` prop set to the value of `state.showConfirmDeleteDialog`)
- education dialog ends up on top
- user clicks 'Yes' in the confirm dialog
- the `onClick` handler bound to the 'Yes' button (in the education dialog) is run, which does something like this:
    - get `state.deletionIndex`
    - `_.cloneDeep(profile)`
    - `profile.education.splice(deletionIndex, 1)`

then there's some cleanup after that. So basically without two actions we don't have enough information to tell whether a user is trying to delete a work history entry or an education entry, and if both pop-ups display we'll try to delete an entry from whichever happens to be on top.

so I think we need to either do what I've got here or something more complicated that places one `ConfirmDeletion` component on `/users`, dispatches an action to differentiate education and work history, and binds another set of callbacks when  a delete icon is clicked.